packages/core/test/generatePolicy.spec.js

```diff
@@ -187,19 +171,86 @@ test('generatePolicy - ESM should flag require/module/exports as global', async
         },
       },
     },
-    'config matches expected'
+    'policy matches expected'
   )
 })
 
-// we no longer throw an error, we log a warning
-// test('generatePolicy - primordial modification', async (t) => {
-//   try {
-//     const config = await createConfigForTest(function () {
-//       const href = window.location.href
-//       Array.prototype.bogosort = () => 'yolo'
-//     })
-//     t.fail('expected to throw an error')
-//   } catch (err) {
-//     t.pass()
-//   }
-// })
+test('trusted root', async (t) => {
```

Comment:
```suggestion
test('trusted root - does not create a resource for $root$', async (t) => {
```

A human language explanation of what the test is about helps when I (re)read it without a fresh memory of what and why.

Reply:
- [ ] rename